### PR TITLE
updated test

### PR DESCRIPTION
**Pull request by janiekyu**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401challenges/src/test/java/code401challenges/ArrayShiftTest.java [hunk 1]
```diff
@@ -17,5 +17,12 @@
                 new int[]{4,8,15,16,23,42},
                 ArrayShift.insertShiftArray(new int[]{4,8,15,23,42}, 16));
     }
+
+    @Test public void testInsertShiftArray_3 () {
+        assertArrayEquals(
+                "Incorrect",
+                new int[]{10,20,30,40,50},
```
Comment (Tw1stedRain) [inline code401challenges/src/test/java/code401challenges/ArrayShiftTest.java]: this test fails. it looks like it is failing due to an incorrect expected output. otherwise this looks good
